### PR TITLE
Updated README. Call next() inside vue route hook to confirm navigation

### PR DESCRIPTION
**Pull request by Fachher**
Call next() inside vue-route hooks (i.e beforeRouteEnter and beforeRouteLeave) to confirm navigation otherwise route is not rendered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,12 +169,14 @@ import Component from 'vue-class-component'
 class MyComp extends Vue {
   // The class component now treats beforeRouteEnter
   // and beforeRouteLeave as Vue Router hooks
-  beforeRouteEnter () {
+  beforeRouteEnter (to: Route, from: Route, next: Function) {
```
Comment (ktsn) [inline README.md]: Please remove type annotations since this snippet is JavaScript.

File: README.md [hunk 2]
```diff
@@ -169,12 +169,14 @@ import Component from 'vue-class-component'
...
     console.log('beforeRouteEnter')
+	next(); // needs to be called to confirm the navigation
```
Comment (ktsn) [inline README.md]: Please follow existing code styling (prefer 2 spaces than hard tab)

**Comment by Fachher** [comment]
Fixed codereview remarks.

**Comment by ktsn** [comment]
Thanks!

**Pull request merged**